app/controllers/api/v1/rooms_controller.rb

```diff
@@ -5,6 +5,7 @@ module V1
     class RoomsController < ApplicationController
       skip_before_action :verify_authenticity_token # TODO: amir - Revisit this.
       before_action :find_room, only: %i[show start]
+      before_action :ensure_authenticated, only: :create
```

Comment:
I would leave this out for now - we should have a separate PR implementing this correctly across all controllers

Reply:
You want me to remove it?